plugins/vendor/zoom/lib/model/ZoomHelper.php

```diff
@@ -101,8 +101,8 @@ public static function loadLoginPage($tokens, $zoomConfiguration)
 		{
 			$page = file_get_contents($file_path);
 			$tokensString = json_encode($tokens);
-			$verificationToken = $zoomConfiguration[kOAuth::VERIFICATION_TOKEN];
-			list($enc, $iv) = AESEncrypt::encrypt($verificationToken, $tokensString);
+			$secretToken = $zoomConfiguration[kOAuth::SECRET_TOKEN];
```

Comment:
to be backward compatible lets check first if we have validation token and if not take the secret token